composer.json

```diff
@@ -18,11 +18,11 @@
         }
     ],
     "require": {
-        "php": "^7.0"
+        "php": "^7.2 || ^8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7 || ^6.0",
-        "friendsofphp/php-cs-fixer": "^2.0"
+        "phpunit/phpunit": "^8.5 || ^9.4",
+        "friendsofphp/php-cs-fixer": "^2.7"
```

Comment:
Let's keep this PR focused on GitHub Actions migrations only, so that I can merge it and see whether everything works as expected. Then we can bump dependency version requirements in a separate PR.

Reply:
Perfect. Let me revert this.

Reply:
I have rolled back the changes in dependencies, and updated the workflow to run against 7.0, 71 and 7.2, which is what was being done in travis.

If everything works, I'll do the dependency updates in next PR.